third_party/istio-1.0.2/prestop-sleep.yaml.patch

```diff
@@ -1,8 +1,6 @@
 349a350,356
->         # PATCH #2: Temporary workaround for https://github.com/knative/serving/issues/2351.
+>         # PATCH #2: Graceful shutdown of the istio-proxy.
 >         lifecycle:
 >           preStop:
 >             exec:
->               command:
->               - /bin/sleep
->               - "20"
+>               command: command: ["sh", "-c", 'until curl -s localhost:15000/clusters | grep "inbound|80|" | grep "rq_active" | grep "rq_active::0"; do sleep 1; done;']
```

Comment:
is this a typo?

if you rerun `download-istio.sh` it will your new patch here.  you may have to change 349a350,356 to 349a350,355 since there is one less line

Reply:
Indeed, fixed it and verified that `download-istio.sh` works now. thanks!